### PR TITLE
feature: Create Spending, add spendings icons

### PR DESCRIPTION
**Pull request by anastasiya-nd**

### CODE REVIEW

File: src/components/SpendingItem/SpendingItem.jsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+import Shopping from '../Icons/Shopping';
+import Entertainment from '../Icons/Entertainment';
+import Car from '../Icons/Car';
+import Bills from '../Icons/Bills';
+import Home from '../Icons/Home';
+import Education from '../Icons/Education';
+import Other from '../Icons/Other';
+import * as Styled from './styles';
+
+
```
Comment (ok2ju) [inline src/components/SpendingItem/SpendingItem.jsx:12]: Redundant space. whats wrong with your vscode?

File: src/components/SpendingItem/SpendingItem.jsx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        return <Education />;
+      case 'other':
+        return <Other />;
+      default:
+        return null;
+    }
+  }
+
+  return (
+    spendings.map(s => (
```
Comment (ok2ju) [inline src/components/SpendingItem/SpendingItem.jsx:67]: `SpendingItem` component should represent one spending but here you trying to render all of them

File: src/components/SpendingItem/SpendingItem.jsx [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+      <Styled.Spending key={s.id}>
+        {getIcon(s.category)}
+        <Styled.CategoryWrap>
+          <Styled.CategoryName>{s.category}</Styled.CategoryName>
+          {s.note && <Styled.Description>{s.note}</Styled.Description>}
+        </Styled.CategoryWrap>
+        {s.labels && (
+          <Styled.LabelWrap>
+            <Styled.Label>{s.labels[0]}</Styled.Label>
```
Comment (ok2ju) [inline src/components/SpendingItem/SpendingItem.jsx:76]: What if there are no labels at all? Event of `s.labels` is empty array this condition always will be true.

**Review by ok2ju — CHANGES_REQUESTED** [review]

File: src/components/SpendingItem/SpendingItem.jsx (hunk 1 above)
Comment (anastasiya-nd) [reply src/components/SpendingItem/SpendingItem.jsx:12]: I don't know.

File: src/components/SpendingItem/SpendingItem.jsx [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+      case 'education':
+        return <Education />;
+      case 'other':
+        return <Other />;
+      default:
+        return null;
+    }
+  }
+
+  const { category, note, labels, createdAt, amount, currency } = props;
```
Comment (ok2ju) [inline src/components/SpendingItem/SpendingItem.jsx:37]: You can use destructuring above:
```js
const SpendingItem = ({ category, note, labels, ... }) => {}
```

File: src/pages/Home/Home.jsx [hunk 5]
```diff
@@ -1,9 +1,54 @@
...
+      note: 'internet, phone, electricity, water',
+      labels: ['Home'],
+      createdAt: 'Sep 22, 2020',
+      amount: 60.50,
+      currency: 'BYN',
+      id: 'fdcfe'
+    },
+  ];
+
+  const spendings = spendingsArray.map((s) => (
```
Comment (ok2ju) [inline src/pages/Home/Home.jsx:35]: Why not to iterate over spendings inside `return`?

**Review by ok2ju — CHANGES_REQUESTED** [review]

**Pull request merged**